configs/prow-dev/config/plugins.yaml

```diff
@@ -12,7 +12,7 @@ config_updater:
 welcome:
   - repos:
       - tidb-community-bots
-    message_template: "Welcome @{{.AuthorLogin}}! <br><br>It looks like this is your first PR to <a href='https://github.com/{{.Org}}/{{.Repo}}'>{{.Org}}/{{.Repo}}</a> 🎉. <br><br>You will be prompted by a bot to use commands during the review process. Do not be afraid to follow the prompts! It is okay to experiment. [Here is the bot commands documentation](https://prow-dev.tidb.io/command-help). <br><br>You can also check if {{.Org}}/{{.Repo}} has [its own contribution guidelines](https://github.com/{{.Org}}/{{.Repo}}/tree/master/CONTRIBUTING.md). <br><br>We want to make sure your contribution gets all the attention it needs! <br><br>Thank you, and welcome to tidb community. :smiley:"
```

Comment:
This welcome is only for tidb-community-bots, so I think the old message is OK for me.

Reply:
I think the sentences like `do not be afraid` and `it is okay to experiment` did not give any useful information but instead disturb the reader.

Reply:
I feel that welcome not only requires useful information, but also a polite and friendly welcome.

Reply:
I did not remove the welcome. I just shortened it and made it more fluent.